.github/workflows/check_dependency_duplications.yml

```diff
@@ -1,39 +1,65 @@
-name: Check npm dependency duplications
-
+name: "npm-dependency-duplications test"
```

Comment:
nitpick: use single quotes for consistency or remove them if they are not needed.